README.md

```diff
@@ -182,20 +190,68 @@ We allow users to specify a `fractionalDigits` option that will display the smal
 #### Example
 
 ```javascript
-new Intl.DurationFormat('en', { fractionalDigits: 2 }).format('PT12.3456S'); // => 12.34 sec
-new Intl.DurationFormat('en', { milliseconds: 'narrow', fractionalDigits: 2 }).format('PT12.3456S'); // => 12s 345.60ms
+const duration = {
+  seconds: 12,
+  milliseconds: 345,
+  microseconds: 600,
+};
+
+// Example using fractionalDigits
+new Intl.DurationFormat("en", { fractionalDigits: 2 }).format(duration);
+// => 12.34 sec
+
+// Example using fractionalDigits and milliseconds set to `narrow`
+new Intl.DurationFormat("en", {
+  milliseconds: "narrow",
+  fractionalDigits: 2,
+}).format(duration);
+// => 12s 345.60ms
```

Comment:
The current spec draft text uses `Intl.ListFormat`, which means the results will look more like:
`12 seconds, 345 milliseconds, and 600 microseconds` and
`12 seconds, 345ms, and 600 microseconds`.